Makefile

```diff
@@ -36,7 +56,9 @@ all build:
 #   make check
 #   make test
 #   make check WHAT=pkg/build GOFLAGS=-v
-check test:
+check-test: export STI_COVER= -cover -covermode=atomic
+check-test: export STI_RACE=  -race
+check-test: verify
```

Comment:
why not just put STI_RACE and STI_COVER inline w/ the hack/test-go invocation?  why 3 targets?